apps/native/app/src/screens/health/health-overview.tsx

```diff
@@ -93,6 +94,7 @@ export const HealthOverviewScreen: NavigationFunctionComponent = ({
 
   const now = useMemo(() => new Date().toISOString(), [])
 
+  const medicinePurchaseRes = useGetMedicineDataQuery()
```

Comment:
is this getting to a point where we should combine all of these queries into one HealthOverview query? 🤔

Reply:
Hmm, it could be. I'll keep it as is for now but revisit if I need to add more than organ donation